README.md

```diff
@@ -8,6 +8,9 @@ A timer component for React that counts down to zero for a specified number of m
  `initialTimeRemaining: Number`
  The time remaining for the countdown (in ms).
 
+`setClass: String (optional -- default: 'timer')`
```

Comment:
I'd prefer to call this `className` for consistency with other React components.